bot.py

```diff
@@ -109,6 +113,64 @@ def get_members_by_role(role):
                 members.append(member)
                 break
     return(members)
+    
+def get_quotes():
+    try:
+        with open(os.path.join(sys.path[0], 'quotes.dat'), 'rb') as fp:
+            return pickle.load(fp)
+    except FileNotFoundError:
+        quotes = []
+        return quotes
+        
+def create_quote_image(quote, name):
+    text = "\"{}\"".format(quote)            
+    name = "- {}".format(name)
+    file = "images/{}.jpg".format(random.randint(1,50))
+    img = Image.open(os.path.join(sys.path[0], file))
+    draw = ImageDraw.Draw(img)
+    img_size = img.size
+    font = ImageFont.truetype("impact.ttf", 180)
+    border = 5
+    multi_line = ""
+    for line in textwrap.wrap(text, width=40):
+        multi_line += line + "\n"
+    text_size = draw.multiline_textsize(text=multi_line, font=font)
+    name_size = draw.textsize(text=name, font=font)
+
+    #Draw quote
+    x = (img_size[0]/2) - (text_size[0]/2)
+    y = (img_size[1]/2) - (text_size[1]/2) - name_size[1]
+    #Border
+    draw.multiline_text((x-border,y),multi_line,font=font, align='center', fill='black')
+    draw.multiline_text((x-border,y-border),multi_line,font=font, align='center', fill='black')
+    draw.multiline_text((x,y-border),multi_line,font=font, align='center', fill='black')
+    draw.multiline_text((x+border,y-border),multi_line,font=font, align='center', fill='black')
+    draw.multiline_text((x+border,y),multi_line,font=font, align='center', fill='black')
+    draw.multiline_text((x+border,y+border),multi_line,font=font, align='center', fill='black')
+    draw.multiline_text((x,y+border),multi_line,font=font, align='center', fill='black')
+    draw.multiline_text((x-border,y+border),multi_line,font=font, align='center', fill='black')
+    #Text
+    draw.multiline_text((x,y),multi_line,font=font, align='center', fill='white')
+
+    #Draw name
+    x += text_size[0] - name_size[0]
+    y += text_size[1]
+    #Border
+    draw.text((x-border,y),name,font=font, align='right', fill='black')
+    draw.text((x-border,y-border),name,font=font, align='right', fill='black')
+    draw.text((x,y-border),name,font=font, align='right', fill='black')
+    draw.text((x+border,y-border),name,font=font, align='right', fill='black')
+    draw.text((x+border,y),name,font=font, align='right', fill='black')
+    draw.text((x+border,y+border),name,font=font, align='right', fill='black')
+    draw.text((x,y+border),name,font=font, align='right', fill='black')
+    draw.text((x-border,y+border),name,font=font, align='right', fill='black')
+    #Text
+    draw.text((x,y),name,font=font, align='right', fill='white')
+
+    timestamp = datetime.datetime.now().strftime('%Y%m%d%H%M%S')
+    out_file = "images/{}.jpg".format(timestamp)
```

Comment:
Lets create use a tmp folder for these generated images. We should also do something about cleaning up files after they've been uploaded.

Reply:
Moved generated images to new tmp folder. They were already being deleted using os.remove().